content/docs/command-reference/gc.md

```diff
@@ -74,14 +73,14 @@ use `--remote` as well.
   that is never referenced from the workspace or from any Git commit can still
   be stored in the project's cache).
 
-  > \* Not including [DVC experiments](
+  > \* Not including [DVC experiments]
 
 [dvc experiments]: /doc/user-guide/experiment-management#experiments
 
 - `--all-experiments` keep cached objects referenced in all [DVC experiments],
   as well as in the workspace (implying `-w`). This preserves the project's
   [experimental](/doc/user-guide/experiment-management) data (including
-  checkpoints).
+  checkpoints). See also `dvc exp gc`.
```

Comment:
This is the only change to this file that is directly related to the task at hand. Sorry, just needed to copy edit the rest when I read it...